noxfile.py

```diff
@@ -90,34 +90,36 @@ def cover(session):
     session.run("coverage", "erase")
 
 
-@nox.session(python="3.8")
+black_pins = ["black==21.12b0", "isort==5.10.1"]
```

Comment:
Something we can do is having the linting dependencies on a requirements file, and enable Dependabot updates? That way, we don't have to stay alert for PyPI updates, and we can focus on checking what Dependabot suggests. Remember that Dependabot will only change pinned versions.

Reply:
The two projects I know of using dependabot for Python (SciPy, manylinux) both dropped it gladly. I personally use it for actions updates, for which it works really well. I don't think I use it for Python anywhere, so I can't address why they didn't like it, though I think it partially was really finicky and not interested in accepting PRs to fix Python issues. I think pre-commit.ci is superior if you'd like to go that way. But not in this PR, I could whip up a pre-commit PR after this one if you'd like to see it.